apps/sms/style/sms.css

```diff
@@ -902,6 +902,18 @@ section[role="region"].new .article-list[data-type="list"]  li {
   pointer-events: none;
 }
 
+/* margin bottom adjustment for inline-block bdi element */
+.suggestion > .name {
+  margin-bottom: -0.5rem;
+}
+
+.suggestion > .name > bdi {
```

Comment:
nit: please, reorder/group css-properties also maybe we can have separate css class for ```bdi``` that supports ellipsis, don't have strong opinion really :) what are your thoughts ?